testdata/jennies/rawtypes/struct_with_complex_fields.txtar

```diff
@@ -297,3 +299,29 @@ func (resource *StringOrBool) UnmarshalJSON(raw []byte) error {
 	return errors.Join(errList...)
 }
 
+-- out/jennies/PythonRawTypes --
+== models/struct_complex_fields.py
+import typing
+
+
+class StructComplexFieldsSomeStructFieldAnonymousStruct:
+    FieldAny: object
+
+
+class SomeStruct:
+    """
+    This struct does things.
+    """
+
+    FieldRef: 'SomeOtherStruct'
```

Comment:
If you import `from __future__ import annotations`, then you can use the actual ref without quoting

Reply:
I tried using that import, but it breaks unions defined as top-level types (with forward references):

```py
# That's broken
ValueMapping = typing.Union[ValueMap | RangeMap | RegexMap | SpecialValueMap]

# That works
ValueMapping = typing.Union['ValueMap' | 'RangeMap' | 'RegexMap' | 'SpecialValueMap']
```

I'm guessing that it's because this import only has an effect on type annotations, and the declaration of `ValueMapping` isn't one.